### PR TITLE
ROC-3482: For no daily interest selected, details of daily interest should not be listed

### PR DESCRIPTION
**Pull request by sabahirfan**
### JIRA link
[ROC-3482](https://tools.hmcts.net/jira/browse/ROC-3482)

### Change description
When claimant has selected no daily interest, we are still displaying the daily interest details with £0 for N days etc. This should not be displayed in this case. Also updated the label from `Interest to date` to `Interest` as advised.

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [x] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/main/views/macro/interestSnippet.njk [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
+            postProcess: 'sprintf', sprintf: [ rate, numberOfDays ] }) }}
+          ({{ t('%s to %s', { postProcess: 'sprintf', sprintf: [ interestDate | date, interestToDate | date ]}) }})
+        </div>
       </div>
-      <div class="font-xsmall">
-        ({{ t('%s to %s', { postProcess: 'sprintf', sprintf: [ interestDate | date, interestToDate | date ]}) }})
+    {% elseif specificDailyAmount > 0 %}
+      <div class="panel text-small" id="details-content-0" aria-hidden>
+        <div>
+          {{ t('Interest calculated with daily interest amount of %s for %s days', {
```
Comment (yarpo) [inline src/main/views/macro/interestSnippet.njk]: Please change it to use new approach. Example can be found here:
src/main/features/dashboard/views/claimant.njk:22

Comment (sabahirfan) [reply src/main/views/macro/interestSnippet.njk]: done

File: src/main/views/macro/interestSnippet.njk [hunk 2]
```diff
@@ -1,23 +1,29 @@
...
-        }}
-        {% else %}
-          {{ t('Interest calculated with daily interest amount of %s for %s days', {
-            postProcess: 'sprintf', sprintf: [ specificDailyAmount | numeral, numberOfDays ] })
-          }}
-        {% endif %}
+    {% if rate %}
+      <div class="panel text-small" id="details-content-0" aria-hidden>
+        <div>
+          {{ t('Interest calculated at {{ rate }}% for {{ numberOfDays }} days', { rate: rate, numberOfDays: numberOfDays}) }}
```
Comment (yarpo) [inline src/main/views/macro/interestSnippet.njk]: <3 ;)

**Comment by pragnesh-patel** [comment]
@geek-soft @rlewan  please could you review this PR please

**Pull request merged**